test/integration/css-fixtures/bad-custom-configuration-arr-3/next.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Refactor the code to NOT use a shared next.config from the parent directory. (Same reason for changes below)